src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -386,6 +382,17 @@ export class Evaluator {
     this.spreadingRelations.removeNode(positionId);
   }
 
+  private buildCompilationParameters() {
+    return buildCompilationParameters(this.context, this.getters, (position) => {
+      const positionId = this.encodePosition(position);
+      const evaluatedCell = this.computeCell(positionId);
+      if (!this.evaluatedCells.has(positionId)) {
+        this.setEvaluatedCell(positionId, evaluatedCell);
+      }
+      return evaluatedCell;
```

Comment:
I'd like to extract the callback to a method, but I don't know how to name it :/

Reply:
computeAndSave ?